App/makersbnb.rb

```diff
@@ -15,17 +15,22 @@ class Makersbnb < Sinatra::Base
   enable :sessions
 
   get '/' do
+    @user = (User.find_by id: session[:user_id])
```

Comment:
I wonder if there's a way to store the user session in some sort of 'configure do' block so its saved for every route? We can save this for another update if not sure.

Reply:
I'll have a look at chitter